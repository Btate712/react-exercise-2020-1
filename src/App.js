import React from 'react';
import User from './components/user/user';
import './App.css';

function App() {
  return (
    <div className="App">
      <User user={{
        name: "Bob", 
        id: 1, 
        posts: [
          {
            title: "Post 1",
            date: "2/2/2020",
            content: "blah, blah, blah"
          },{
            title: "Post 2",
            date: "2/3/2020",
            content: "more words..."
          },{
            title: "Post 3",
            date: "2/4/2020",
            content: "getting tired of making this stuff up..."
          }
        ]
      }}/>
    </div>
  );
}

export default App;
