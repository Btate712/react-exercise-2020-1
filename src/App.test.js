import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer';


afterEach(cleanup);

test('Renders app', () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
