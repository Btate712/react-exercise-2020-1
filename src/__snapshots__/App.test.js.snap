// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders app 1`] = `
<div
  className="App"
>
  <h1>
    User Name: 
    Bob
  </h1>
  <h2>
    User ID: 
    1
  </h2>
  <h2>
    Posts:
  </h2>
  <ul>
    <li>
      Post 1
       - 
      2/2/2020
    </li>
    <li>
      Post 2
       - 
      2/3/2020
    </li>
    <li>
      Post 3
       - 
      2/4/2020
    </li>
  </ul>
</div>
`;
